.github/workflows/test.yml

```diff
@@ -13,7 +13,7 @@ jobs:
           java-version: '12.x'
       - uses: subosito/flutter-action@v1
         with:
-          channel: 'beta'
+          channel: 'stable'
```

Comment:
changed this just for this branch. since pub uses stable we should make sure our build will be green on pub.